### PR TITLE
add pie-chart v1.0.0

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/n3-charts/pie-chart
**Watch 14
  Star 165
 Fork 51**
@Amomo could u help me check this PR for #6769 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update) 
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/pie-chart/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/n3-charts/pie-chart.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/n3-charts/pie-chart.git",
+    "basePath": "dist",
+    "files": [
+      "!(pie-chart.spec.js)"
```
Comment (PeterDaveHello) [inline ajax/libs/pie-chart/package.json]: What's the purpose of this file?

Comment (dimitrismistriotis) [inline ajax/libs/pie-chart/package.json]: https://github.com/n3-charts/pie-chart/blob/master/dist/pie-chart.spec.js

**Comment by PeterDaveHello** [comment]
Ok, let's rebase this branch with our latest master branch and them merge it.

**Comment by Amomo** [comment]
@maruilian11 sorry for late, could you rebase this branch with our latest master branch again?

**Pull request merged**